modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -498,6 +498,12 @@ AUI.add(
 					if (
 						currentTarget.hasClass('lfr-ddm-repeatable-add-button')
 					) {
+						var field = instance.getFieldDefinition();
+
+						if (field.type === 'select' && field.options[0].value === '') {
```

Comment:
Hi @matthewchan1,

Why did you need to and this extra check `field.options[0].value === ''`?

Reply:
Hi @natocesarrego, the extra check is insurance in case the call to repeat() is triggered before Select Field's getOptions(), which creates the placeholder with ''. It's a very unlikely edge case since rendering the page uses getOptions, so if you'd like, we can remove that added check.

Reply:
If you don't find a valid use case for it, @matthewchan1, then I believe we should remove it. Otherwise, keep it. In addition, can you please rebase and resend due to conflicts? Thanks.